c/certificateService.c

```diff
@@ -109,110 +91,86 @@ static void respondWithInvalidMethod(HttpResponse *response) {
     finishResponse(response);
 }
 
-static void respondWithBadRequest(HttpResponse *response) {
-    jsonPrinter *p = respondWithJsonPrinter(response);
-
-    setResponseStatus(response, 400, "Bad Request");
-    setDefaultJSONRESTHeaders(response);
-    writeHeader(response);
-
-    jsonStart(p);
-    {
-      jsonAddString(p, "error", "The length of the certificate is longer than 4096 bytes");
-    }
-    jsonEnd(p);
-
-    finishResponse(response);
-}
-
-static int serveMappingService(HttpService *service, HttpResponse *response)
-{
+static int serveMappingService(HttpService *service, HttpResponse *response) {
   HttpRequest *request = response->request;
 
   if (!strcmp(request->method, methodPOST))
   {
-    RUsermapParamList *userMapCertificateStructure 
-      = (RUsermapParamList*)safeMalloc31(sizeof(RUsermapParamList),"RUsermapParamList");
-    memset(userMapCertificateStructure, 0, sizeof(RUsermapParamList));
 
-    if(request->contentLength > sizeof(userMapCertificateStructure->certificate) || request->contentLength < 0) {
-      respondWithBadRequest(response);
-      return 0;
+    int urlLength = strlen(request->uri);
+    if(urlLength < 0 || urlLength > 17) {
```

Comment:
Why 17? I suggest you define this value (i.e. via a `#define`).

Reply:
changed to macro